### DISCUSSION TITLE
How to download the compiled (bundled) files from a react sandbox

### DISCUSSION THREAD

**Discussion by jsejcksn** [codesandbox/codesandbox-client#5133] [Q&A] [answered]
I can download my source files, but how do I download the contents of the build directory?

**Comment by SaraVieira** [comment, marked as answer]
Hey!

Our build is made specifically for CodeSandbox so it would not work locally that's when you deploy to Vercel or netlifly we build it using react scripts instead of our own bundler since ours turns things like `import` to `$csb_import` that are internal functions.

We do not currently make it available to download since to work on a machine or deployment service you will need to build using the appropriate building tool

**Reply by jsejcksn** [reply]
@SaraVieira The build happens client-side, in my browser, right?

**Reply by SaraVieira** [reply]
yes, that is correct.

**Reply by jsejcksn** [reply]
@SaraVieira Thanks.

> that's when you deploy to Vercel or netlifly we build it using react scripts instead of our own bundler

**So how do I get these files for use on my own local machine?**

Basically, I'm looking for "deploy to localhost". "Deploy to GitHub pages" would also be a great option.

I haven't thoroughly researched it, but I think you'd just need to support the `.env` file with the "[advanced configuration](https://create-react-app.dev/docs/advanced-configuration/)" options and [the `homepage` property](https://create-react-app.dev/docs/deployment#building-for-relative-paths) in `package.json`.

**Reply by jsejcksn** [reply]
Since CodeSandbox is already building my project in the browser, it is a huge pain to have to

- download the sandbox zip
- decompress the sandbox zip
- install node and npm
- install all the npm deps in package.json
- run the react scripts build script

just to get a static site bundle for use on my `localhost` server

**Reply by SaraVieira** [reply]
I'm sorry but this is not an option since we don't build a version you can use locally or on Github pages, if we did build for GitHub pages we would build it differently before.
